sample_reponse.json

```diff
@@ -48,476 +40,724 @@
-                                                "2021-10-15T09:26:11.000Z",
-                                                "2021-10-15T10:36:11.000Z",
-                                                "2021-10-15T11:46:11.000Z",
-                                                "2021-10-15T12:56:11.000Z",
-                                                "2021-10-15T14:06:11.000Z",
-                                                "2021-10-15T15:15:26.000Z",
-                                                "2021-10-15T16:26:07.000Z",
-                                                "2021-10-15T02:42:02.000Z",
-                                                "2021-10-15T03:53:36.000Z",
-                                                "2021-10-15T05:03:41.000Z",
-                                                "2021-10-15T06:13:41.000Z",
-                                                "2021-10-15T07:23:41.000Z",
-                                                "2021-10-15T08:33:41.000Z",
-                                                "2021-10-15T09:43:41.000Z",
-                                                "2021-10-15T10:53:41.000Z",
-                                                "2021-10-15T12:03:41.000Z",
-                                                "2021-10-15T13:13:41.000Z",
-                                                "2021-10-15T14:25:33.000Z",
-                                                "2021-10-15T03:18:55.000Z",
-                                                "2021-10-15T04:28:41.000Z",
-                                                "2021-10-15T05:38:41.000Z",
-                                                "2021-10-15T06:48:41.000Z",
-                                                "2021-10-15T07:58:41.000Z",
-                                                "2021-10-15T09:08:41.000Z",
-                                                "2021-10-15T10:18:41.000Z",
-                                                "2021-10-15T11:28:41.000Z",
-                                                "2021-10-15T12:38:41.000Z",
-                                                "2021-10-15T13:48:41.000Z",
-                                                "2021-10-15T15:01:11.000Z",
-                                                "2021-10-15T16:11:36.000Z"
-                                            ]
-                                        }
+                            "location_id": "EDAP",
+                            "fulfillment_id": "EDAP_TO_ALVA",
+                            "matched": true
+                        }
+                    ],
+                    "fulfillments": [
+                        {
+                            "id": "CGPP_TO_ALVA",
+                            "start": {
+                                "location": {
+                                    "id": "CGPP"
+                                },
+                                "time": {
```

Comment:
Incorrect representation of `Fulfillment` object. Kindly refer to my responses on the linked Issue #1

Reply:
Also remember that consumer could be looking for a "day pass" which does not need any start/end points to be specified. Just making sure we are not eliminating use cases such as those.

Reply:
Could this be represented as an item object with fulfillment_id referring to an object like shown below :
```
"items": [
    {
        "id": "PAY_PASS",
        "descriptor": {
            "name": "Day Pass Ticket",
            "code": "DPT"
        },
        "price": {
            "currency": "INR",
            "value": "100"
        },
        "matched": true
    }
],
"fulfillments" : [
    {
        "id": "DAY_PASS_FULFILLMENT",
        "start": {
            "time": {
                "timestamp": "2022-02-13T06:30:00.000Z"
            }
        },
        "end": {
            "time": {
                "timestamp": "2022-02-14T06:30:00.000Z"
            }
        }
    }
]
```